### PR TITLE
doc: vm with k8s and nri-jmx configuration using annotations

### PR DESCRIPTION
**Pull request by rubenruizdegauna**

### CODE REVIEW

**Comment by rogercoll** [comment]
@rubenruizdegauna Great work!

Can we format the READMEs? They seem not to have a proper structure.

In addition, what do you think if we configure the `NRIA_STAGING` env from a ConfigMap or a Secret like the NRIA_LICENSE_KEY?

**Comment by rubenruizdegauna** [comment]
> @rubenruizdegauna Great work!
> 
> Can we format the READMEs? They seem not to have a proper structure.
> 
> In addition, what do you think if we configure the `NRIA_STAGING` env from a ConfigMap or a Secret like the NRIA_LICENSE_KEY?

thanks for the feedback :)

changes in https://github.com/newrelic/nri-jmx/pull/90/commits/aa92bd9b5f8bba23c839d57fba1b93d060c9fdee

**Review by rogercoll — APPROVED** [review]
LGTM

File: docs/vms/jmx_in_k8s/provision/create-secret.sh.dist [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+NR_LICENSE=""
+
+microk8s kubectl delete secret secrets 2>/dev/null
+microk8s kubectl create secret generic secrets --from-literal=nriaLicenseKey="${NR_LICENSE}"
```
[the remark this answers is not in the document]
Comment (rubenruizdegauna) [reply docs/vms/jmx_in_k8s/provision/create-secret.sh.dist:6]: I added it to the dist file https://github.com/newrelic/nri-jmx/pull/90/files#diff-6f9996c2cbf75e07623600102d647cd8ab495eb82f2b40dab8db21b00963f939 among the cluster name in order to have just one configMap, and added it to the README to be clear. I'm merging it to move forward and not spend too much time on this :)

**Pull request merged**